CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+
+# Environnement de développement
+
+## Outils de développement nécessaires
+
+- PHP
+- [composer](https://getcomposer.org)
+- NodeJS
+- [yarn](https://yarnpkg.com)
+- [PostgreSQL](https://www.postgresql.org)
+
+Alternativement, il est possiblement d'utiliser uniquement Docker.
+
+## Installation
+
+    composer install --prefer-dist --no-interaction --no-ansi --no-progress --no-suggest
+    yarn install && yarn build
+
+## Lancement en local
+
+    yarn encore dev --watch
+
+Et dans un autre terminal :
+
+    php bin/console server:run
+
+## Exécution des tests unitaires
+
+    php bin/console doctrine:migrations:migrate -n
+    php bin/console doctrine:fixtures:load -n
+    php bin/phpunit tests/Controller/ProcessControllerTest.php
```

Comment:
Typo with the filename here :)

Reply:
Why that ? The file do exist : [tests/Controller/ProcessControllerTest.php](https://github.com/tgalopin/enpremiereligne.fr/blob/master/tests/Controller/ProcessControllerTest.php)

Reply:
I meant that I wouldn't limit the PHPUnit run to this file but would run all the tests :) .

Reply:
You're probably right, I may have installed the wrong tool.
Do you have the same output ?
```
$ symfony --version
Symfony Installer version 1.5.11
```

Reply:
Alright ! I modified this